### PR TITLE
 [MIG] website_field_autocomplete: Migration to 11.0

### PR DESCRIPTION
**Pull request by bouvyd**

### CODE REVIEW

File: website_field_autocomplete/demo/field_autocomplete_demo.xml
Comment (yajo) [inline website_field_autocomplete/demo/field_autocomplete_demo.xml]: Just curious... why are you using a `<record>` instead of the previous `<template>`?

**Review by yajo — CHANGES_REQUESTED** [review]
Please squash all translation commits together including 8ba7a43 and make bots go 💚 😊

File: website_field_autocomplete/demo/field_autocomplete_demo.xml
Comment (bouvyd) [reply website_field_autocomplete/demo/field_autocomplete_demo.xml]: Because of changes in the `website` module, notably odoo/odoo@4ecbacaf59576a22ff45615a5aa5c67244e4fb93

You can no longer use a `page=True` attribute on the template definition, you have to declare a view & page now (at least that's what I understand of these changes - I took example on [`website` data files](https://github.com/odoo/odoo/blob/ba9381ed2a4765dc5f5f8d18c0e13085ba717815/addons/website/data/website_data.xml#L33-L51))

**Comment by nhomar** [comment]
@dbo-odoo Thanks for push your work to the OCA.

I like what I see, just wonder if th ered on runbot can be fixed, but falling appart of that it is a +1 for me.

**Comment by bouvyd** [comment]
Not sure why the runbot is yellow.

Isn't this a runbot problem?
```
<string>:20: (WARNING/2) Cannot analyze code. Pygments package not found.
```

I'll check the travis bot tomorrow, probably :)

**Comment by pedrobaeza** [comment]
Maybe you need to add `Pygments` to requirements.txt file in root for being installed.

File: website_field_autocomplete/__manifest__.py
Comment (simahawk) [inline website_field_autocomplete/__manifest__.py]: 10 -> 11 ;)

File: website_field_autocomplete/controllers/main.py
Comment (simahawk) [inline website_field_autocomplete/controllers/main.py:20]: routes' methods shouldn't be public methods?

**Review by simahawk — CHANGES_REQUESTED** [review]

File: website_field_autocomplete/controllers/main.py
Comment (bouvyd) [reply website_field_autocomplete/controllers/main.py:20]: They don't have to be as far as I'm aware. I see no need to change this as part of a migration, since I'm not the original author I don't know if it was a conscious choice or not...

**Comment by bouvyd** [comment]
I've made changes following reviews by @Yajo and @simahawk 

I have still no idea how codecov works of if I'm supposed to do anything about it...

**Review by Daemo00 — APPROVED** [review]
Can't manage to have a runbot for this, tried in local and it is ok.
Code looks good, please consider https://github.com/dbo-odoo/website/pull/2

**Comment by rafaelbn** [comment]
Hello! @Yajo @simahawk could you unblock this PR? It looks like @dbo-odoo attend your comments.

@dbo-odoo could you merge https://github.com/dbo-odoo/website/pull/2 ?

**Review by yajo — APPROVED** [review]

File: website_field_autocomplete/README.rst
Comment (nikul-serpentcs) [inline website_field_autocomplete/README.rst:53]: @dbo-odoo Change Target `11.0`

File: website_field_autocomplete/__init__.py
Comment (nikul-serpentcs) [inline website_field_autocomplete/__init__.py:2]: IMO Remove Copyright from all `__init__` file

File: website_field_autocomplete/__manifest__.py
Comment (nikul-serpentcs) [inline website_field_autocomplete/__manifest__.py:10]: Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Changes

**Pull request closed without merging**
